ceremonies/src/lib.rs

```diff
@@ -447,7 +447,11 @@ impl<T: Trait> Module<T> {
                 .chain(endorsees.into_iter())
                 .chain(newbies.into_iter());
 
-            let n_meetups = n / 12 + 1;
+            let mut n_meetups = n / 12;
+            if n.rem_euclid(12) > 0 {
```

Comment:
Please read #4 carefully! This commit does NOT fix #4. You need to fetch the number of locations defined for this community and limit the number of meetups to that number!

Reply:
Oh, sorry for that. I did indeed not read it carefully enough.